files/install_graphdb.sh

```diff
@@ -11,7 +11,7 @@ timedatectl set-timezone UTC
 
 # Install Tools
 apt-get -o DPkg::Lock::Timeout=300 update -y
-apt-get -o DPkg::Lock::Timeout=300 install -y bash-completion jq nvme-cli openjdk-11-jdk unzip
+apt-get -o DPkg::Lock::Timeout=300 install -y bash-completion jq nvme-cli openjdk-21-jdk unzip
```

Comment:
@yaskoo, should we switch to Adoptium's Temurin JRE?

Reply:
wouldn't hurt I guess, since we are using it the container images